### DISCUSSION TITLE
[Langchain] .msg from MS Outlook only generating strange symbols (UnstructuredAPIFileLoader¶)

### DISCUSSION THREAD

**Discussion by guidorietbroek** [Unstructured-IO/unstructured#1151] [Q&A]
When using the UnstructuredAPIFileLoader[¶] from Langchain .msg files are not generating plain text but strange symbols. Could be something with encoding?

```
from langchain.document_loaders import UnstructuredEmailLoader

email_loader = UnstructuredEmailLoader(
    file_path="documents/ANY_EMAIL.msg",
    mode='elements',
    process_attachments=False
)

email = email_loader.load()
```

**Comment by Coniferish** [comment]
Hey @guidorietbroek! Thanks reaching out. I'll take a look into this for you.

**Reply by Coniferish** [reply]
@guidorietbroek I'm not able to reproduce your issue. Can you provide more detail such as a sample email that's creating this issue? If there is a problem, I'll open an issue for you and get to work on it.
